### DISCUSSION TITLE
installation issues

### DISCUSSION THREAD

**Discussion by PARobertson** [qcxms/QCxMS#66] [Q&A] [answered]
Hi, 

I'm excited to get started using QCxMS, but am having trouble getting it installed.

I am using PyCharm on Windows 10. I have successfully run conda config --add channels conda-forge, but when I try to install qcxms it gives me a "failure to solve environment" and PackagesNotFoundError: 

My colleagues and I have tried a bunch of default troubleshooting options, but haven't had any luck.

Has anyone encountered this issue before, or know how to get around it?

Cheers,
Patrick

**Comment by JayTheDog** [comment, marked as answer]
Hi, 
so I checked if the the repository can be found using `conda search qcxms --channel conda-forge`. 
For me it shows the following:
```
Loading channels: done
# Name                       Version           Build  Channel             
qcxms                          5.1.3      hb97063f_0  conda-forge         
qcxms                          5.1.3      hb97063f_1  conda-forge         
qcxms                          5.2.0      h92c58b1_0  conda-forge 
```
But this is done at a Linux machine in a terminal. As the entire QCxMS program is developed for unix based machines, I'm pretty sure that if you would find a way to install it through `conda` in any form, the program won't work under Windows anyways, sorry. The best solution would probably to install a VM from which you could access a Linux distro, but I don't know how the parallelization would work out on such a setup.

**Reply by PARobertson** [reply]
Thanks for that. 

I had sort of come to the same conclusions myself and have installed it via a VM. However, as you forsaw, there are issues with parallelization. I've contacted our HPC team and hopefully they let me install it on our group cluster.

**Comment by JayTheDog** [comment]
A working example on Windows was provided in this discussion before:
https://github.com/qcxms/QCxMS/discussions/32
